TWLight/urls.py

```diff
@@ -53,6 +53,7 @@
     ),
     # Third-party --------------------------------------------------------------
     url(r"^comments/", include("django_comments.urls")),
+    url(r"^admin/rosetta/", login_required(include("rosetta.urls"))),
```

Comment:
I don't think rosetta really belongs in our app since it can't be used in our production workflows. Translations for version-controlled messages need to happen in translatewiki.